test/IDE/complete_after_super.swift

```diff
@@ -440,11 +442,16 @@ class SuperDerivedB : SuperBaseB {
 
   class func test3() {
     super#^CLASS_FUNC_SUPER_NODOT^#
+  }
+  class func test4() {
+    super.#^CLASS_FUNC_SUPER_DOT^#
+  }
+  class func test5() {
     super(#^CLASS_FUNC_SUPER_PAREN^#
-
-// CLASS_FUNC_SUPER_NODOT: Decl[Constructor]/CurrNominal: .init()[#SuperBaseB#]
-// CLASS_FUNC_SUPER_NODOT: Decl[Constructor]/CurrNominal: .init({#a: Double#})[#SuperBaseB#]
-// CLASS_FUNC_SUPER_NODOT: Decl[Constructor]/CurrNominal: .init({#int: Int#})[#SuperBaseB#]
+  }
+// CLASS_FUNC_SUPER: Decl[Constructor]/CurrNominal: {{.init|init}}()[#SuperBaseB#]
+// CLASS_FUNC_SUPER: Decl[Constructor]/CurrNominal: {{.init|init}}({#a: Double#})[#SuperBaseB#]
+// CLASS_FUNC_SUPER: Decl[Constructor]/CurrNominal: {{.init|init}}({#int: Int#})[#SuperBaseB#]
   }
```

Comment:
This brace is now unmatched after your changes (it was originally matching the one in `test3() {`).

Reply:
Indeed, well noticed